modules/900-kubevirt/cdi-ing.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: cdi-uploadproxy
```

Comment:
What is this?

Reply:
my bad, this is from another PR.
removed